docs/kratos/bring-your-own-ui/20_configure-ory-to-use-your-ui.mdx

```diff
@@ -3,20 +3,88 @@ id: configure-ory-to-use-your-ui
 title: Configure Ory to use your UI
 ---
 
-Unless you are building a native app, you must configure the endpoints where your UI pages for the individual flows are located.
+To define the paths where Ory calls your custom UI, go to [**Ory Console**](https://console.ory.sh/) → **User Interface**. This
+view allows you to define the URLs of UI views for every self-service flow. To point to the desired UI, you can use relative or
+absolute paths.
 
-To define the endpoints where Ory should look for your custom UI, go to [**Ory Console**](https://console.ory.sh/) → **User
-Interface**. If you don't define any of the endpoints, Ory Account Experience will be used for that flow.
+:::info
 
-Let's say you have an app hosted at `https://www.example.org` with a login screen at `https://www.example.org/login` but you don't
-have other screens implemented yet. It's okay! Provide the login UI endpoint and leave the other endpoints untouched to use Ory
-Account Experience.
+Unless you set a custom UI base URL, relative links always use your SDK Configuration URL as the base. To check your SDK
+Configuration URL, go to [**Ory Console**](https://console.ory.sh/) → **Access & APIs**.
+
+:::
+
+By default, self-service flows use [Ory Account Experience](./01_overview.mdx#why-should-i-use-ory-account-experience), which is a
+part of every Ory Network project. In the default setup, the system uses relative paths to point to the appropriate UI for every
+screen. The relative links that point to the Ory Account Experience follow the `/ui/{flow_name}` format.
+
+## Custom domain setup
+
+All paid Ory Network plans allow to add a custom domain to your project. After adding a custom domain, you can also set up a base
+URL for all custom UI views. This is a great way to make Ory work with your existing deployment without adjustments, as you can
+use the combination of the base URL and relative URLs to point Ory to UI screens for operations such as login or registration you
+are already hosting.
+
+To set the base URL for custom UI and define the paths for UI views, go to [**Ory Console**](https://console.ory.sh/) → **User
+Interface**
+
+:::caution
+
+When you define a custom UI base URL, you lose access to the Ory Account Experience. The custom UI base url takes precedence over
+SDK Configuration URL.
```

Comment:
Just note that the SDK url or Project url becomes the custom domain name "auth.acme.com". It will always auto redirect to this domain of one is set.